npc/custom/breeder.txt

```diff
@@ -14,20 +14,20 @@ prontera,124,201,1	script	Universal Rental NPC	726,{
 	if (ismounting()) {
 		message strcharinfo(0),"You must first remove your mount.";
 		end;
-	} else if (((eaclass()&EAJ_THIRDMASK)==EAJ_RANGER) && !countitem(6124)) {
+	} else if (((eaclass()&EAJ_THIRDMASK)==EAJ_RANGER || (eaclass()&EAJ_THIRDMASK)==EAJ_WINDHAWK) && !countitem(6124)) {
 		if (!checkfalcon() && getskilllv("HT_FALCON") && !checkwug()) {
 			if(select(" ~ Falcon: ~ Warg")==1) setfalcon;
 			else getitem 6124,1; //Wolf's_Flute
 		} else getitem 6124,1; //Wolf's_Flute
-	} else if (((eaclass()&EAJ_THIRDMASK)==EAJ_MECHANIC) && !checkcart() && getskilllv("MC_PUSHCART")) {
+	} else if (((eaclass()&EAJ_THIRDMASK)==EAJ_MECHANIC || (eaclass()&EAJ_THIRDMASK)==EAJ_MEISTER) && !checkcart() && getskilllv("MC_PUSHCART")) {
```

Comment:
so this is must change to? As suggested by @Lemongrass3110

Reply:
My comment applies to this whole file. All this changes should be reverted.